source/client-side-encryption/client-side-encryption.rst

```diff
@@ -976,6 +976,13 @@ being created. If ``keyMaterial`` is given, the custom key material is used for
 encrypting and decrypting data. Otherwise, the key material for the new data key
 is generated from a cryptographically secure random device.
 
+rewrapManyDataKey
+-----------------
+
+If applicable, drivers MUST document users must upgrade dependencies if necessary to avoid being impacted by MONGOCRYPT-464.
+
+If applicable, drivers MUST return an error if rewrapManyDataKey is called with libmongocrypt 1.5.1 or older.
```

Comment:
rewrapManyDataKey was new in 1.5.0, right? If so, say 1.5.1 or 1.5.0 instead of 1.5.1 or older.

Reply:
Yes. Done.